.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Report a bug to help us improve.
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce
+      description: Please provide a *minimal* script to reproduce the bug.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Error Output
+      description: Please indicate the output from the above script and what you expect the output to be.
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: freud Version
+      description: What version of freud are you using?
+      placeholder: v2.10.0
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Python Version
+      description: What version of python are you using?
+      placeholder: v3.8.2
```

Comment:
Given how many people are running on supercomputing clusters with arbitrarily slow version updating, I wouldn't include a default here.

Reply:
AFAIK, the placeholder isn't a default option, its something that is there to show users how to format their response. Try filing a hoomd issue so you can see how these look.

Reply:
Thanks for that, I didn't realize how the placeholder works. I see it now.